Mathlib/Analysis/BoxIntegral/UnitPartition.lean

```diff
@@ -293,4 +308,183 @@ theorem prepartition_isPartition {B : Box ι} (hB : hasIntegralVertices B) :
   rw [TaggedPrepartition.mem_toPrepartition, mem_prepartition_iff]
   exact ⟨index n x, mem_admissibleIndex_of_mem_box n hB hx, rfl⟩
 
+theorem exists_admissibleIndex {B : Box ι} (hB : hasIntegralVertices B) {x : ι → ℝ} (hx : x ∈ B) :
+    ∃ ν ∈ admissibleIndex n B, box n ν = box n (index n x) :=
+  ⟨index n x, mem_admissibleIndex_of_mem_box n hB hx, rfl⟩
+
+open Submodule Pointwise BigOperators
+
+open scoped Pointwise
+
+variable (c : ℝ) (s : Set (ι → ℝ)) (F : (ι → ℝ) → ℝ)
+
+local notation "L" => span ℤ (Set.range (Pi.basisFun ℝ ι))
```

Comment:
Is this the standard lattice (like ℤⁿ ⊆ ℝⁿ)? It would perhaps be good to add a docstring.